e2e/pages/app/toolbar-search/toolbar-search.testpage.component.html

```diff
@@ -1,83 +1,98 @@
+                    <ux-icon name="search"></ux-icon>
+                </button>
 
-    <div class="demo-toolbar-right" [formGroup]="form">
-        <button type="button" class="btn btn-link btn-icon button-secondary">
-            <ux-icon name="grid"></ux-icon>
-        </button>
+            </ux-toolbar-search>
 
-        <button type="button" class="btn btn-link btn-icon button-secondary">
-            <ux-icon name="list"></ux-icon>
-        </button>
+            <button type="button" class="btn btn-link btn-icon button-secondary">
+                <ux-icon name="add"></ux-icon>
+            </button>
 
-        <ux-toolbar-search
-            id="searchRight"
-            [alwaysExpanded]="alwaysExpanded"
-            direction="left"
-            background="white"
-            (search)="onSearchRight()">
+            <button type="button" class="btn btn-link btn-icon button-secondary">
+                <ux-icon name="edit"></ux-icon>
+            </button>
+        </div>
 
-            <input uxToolbarSearchField
-                formControlName="search"
-                type="text"
-                class="form-control"
-                placeholder="Search"
-                aria-label="Search">
+        <div class="demo-toolbar-right" [formGroup]="form">
+            <button type="button" class="btn btn-link btn-icon button-secondary">
+                <ux-icon name="grid"></ux-icon>
+            </button>
 
-            <button uxToolbarSearchButton
-                    type="button"
-                    class="btn btn-link btn-icon button-secondary"
-                    aria-label="Toggle Search">
-                <ux-icon name="search"></ux-icon>
+            <button type="button" class="btn btn-link btn-icon button-secondary">
+                <ux-icon name="list"></ux-icon>
             </button>
-        </ux-toolbar-search>
+
+            <ux-toolbar-search
+                id="searchRight"
+                [alwaysExpanded]="alwaysExpanded"
+                direction="left"
+                background="white"
+                (search)="onSearchRight()">
+
+                <input uxToolbarSearchField
+                    formControlName="search"
+                    type="text"
+                    class="form-control"
+                    placeholder="Search"
+                    aria-label="Search">
+
+                <button uxToolbarSearchButton
+                        type="button"
+                        class="btn btn-link btn-icon button-secondary"
+                        aria-label="Toggle Search">
+                    <ux-icon name="search"></ux-icon>
+                </button>
+            </ux-toolbar-search>
+        </div>
+
     </div>
 
-</div>
+    <p class="m-t-md">Searched for:
+        <strong id="searchedFor">{{searchedFor}}</strong>
+    </p>
 
-<p class="m-t-md">Searched for:
-    <strong id="searchedFor">{{searchedFor}}</strong>
-</p>
+    <div id="mastheadSearchText" class="m-t-md">
```

Comment:
I don't think this is required, this is testing the selection functionality of a typeahead, which should be covered by the typeahead tests.